en/getting_started/sensor_selection.md

```diff
@@ -1,61 +1,60 @@
-pressure sensor for proper altitude reading. Despite that, we strongly
-recommend using **GPS** in all frames in order to unveil the full
-potential of the aircraft or vehicle and enable its automatic modes. For
-most of the fixed wing aircraft, **Airspeed sensor** is mandatory as
-well.
+PX4-based systems use sensors to determine vehicle state (state is required for stabilization and to enable autonomous control). The sensors provide information including: position/altitude, heading, speed, airspeed, orientation (attitude) and rates of rotation in different directions.
+
+The system *minimally requires* a gyroscope, accelerometer, magnetometer (compass) and barometer. A GPS or other positioning system is needed to enable all automatic [modes](../getting_started/flight_modes.md#categories), and some assisted modes. Fixed wing and VTOL-vehicles should additionally include an airspeed sensor (very highly recommended).
+
+The minimal set of sensors is incorporated into [Pixhawk Series](../flight_controller/pixhawk_series.md) flight controllers (and may also be in other controller platforms). Additional/external sensors can be attached to the controller.
+
+Below we describe some of the sensors. At the end there are links to information about [sensor wiring](#wiring).
+
 
 ## GPS
 
-There are many types of GPS you can use with Pixhawk and PX4. We
-recommend using GPS + Compass solution like the original 3DR GPS or
-alternative LEA 6H, NEO 7M, NEO 8 GPS+Compass solutions.
+Many types of GPS can be used with PX4. When using Pixhawk-series flight controllers we recommend using a *combined GPS + Compass*.
+
+> **Tip** The Pixhawk internal compass is close to other electronics, and may be susceptible to electromagnetic noise. An external compass/GPS mounted on a pedestal or wing (for fixed-wing) away from the rest of the system is recommended.
+
+Options include the original 3DR GPS, LEA 6H, NEO 7M, NEO 8 GPS+Compass solutions.
 
 ![3DR GPS_Top](../../images/gps_3dr_top_1.jpg) 
 ![GPS + Compass](../../images/gps_compass.jpg)
 
-Because the internal compass of Pixhawk is close to other electronics it
-could be susceptible to electromagnetic noise. So when we use the
-external Compass we can locate it away from electronics on a pedestal or
-at the wing of the plane.
 
 ## Airspeed
 
-The Airspeed sensor is mandatory for most of the Airplane Frames and for
-all VTOL Frames. It is very important sensor because the PX4 autopilot
-does not have other means to detect stall, except with the airspeed
-sensor. For the airplane it is the airspeed that guarantees lift not
-ground speed. We recommend using the digital airspeed sensor.
+The Airspeed sensor is required for most of fixed-wing frames and for all VTOL frames.
+
+This sensor is so important because the autopilot does not have other means to detect stall. 
+For fixed-wing vehicles it is the airspeed that guarantees lift not ground speed!
+
+We recommend using a digital airspeed sensor.

```

Comment:
What airspeed sensor. Can you provide list/links to some valid options? This will all be easier when we add docs into our drivers :-)

Reply:
I'm not really familiar with these, but here's one: https://store.mrobotics.io/mRo-I2C-Airspeed-Sensor-JST-GH-p/mro-classy-arspd-mr.htm